src/lib/snarky/src/curves.ml

```diff
@@ -888,6 +888,8 @@ module Make_weierstrass_checked
        = b * t
     *)
     let open Let_syntax in
+    (* TODO SNARK optimization:
+        Can get away with using an unsafe add if we modify this a bit. *)
```

Comment:
We've been trying to put issue numbers on all new TODOs since open-sourcing, can you make an issue for this and do `TODO #xxx : Can we...`